app/assets/javascripts/miq_application.js

```diff
@@ -1383,3 +1383,9 @@ function miqInitMainContent() {
 
   $('#main-content').css('height', 'calc(100% - ' + height + 'px)')
 }
+
+function expanded_td(link) {
+    var link = $(link);
+    link.find("i").toggleClass("glyphicon-plus").toggleClass("glyphicon-minus");
```

Comment:
These 2 toggleClass calls can be replaced with one: `link.find("i").toggleClass("glyphicon-plus glyphicon-minus"))`.

[JQuery docs](http://api.jquery.com/toggleclass/):

> This method takes **one or more class names** as its parameter. In the first version, if an element in the matched set of elements already has the class, then it is removed; if an element does not have the class, then it is added.